prog/dftb+/lib_io/xmlutils.F90

```diff
@@ -221,6 +222,74 @@ recursive subroutine getTagsWithoutAttr_recursive(node, name, rootOnly, &
 
   end subroutine getTagsWithoutAttr_recursive
 
```

Comment:
The routine is OK, but as far as I can see, it is not used anywhere. Could we wait adding it, until it is really needed?